### PR TITLE
arm IT AL fix

### PR DESCRIPTION
**Pull request by WorksButNotTested**

### CODE REVIEW

File: gum/arch-arm/gumthumbrelocator.c [hunk 1]
```diff
@@ -911,6 +911,11 @@ gum_thumb_relocator_rewrite_it_block_start (GumThumbRelocator * self,
 
   block->then_label = self->output->code + 1;
   block->end_label = NULL;
+  block->cc = cc;
+  block->active = TRUE;
```
Comment (oleavr) [inline gum/arch-arm/gumthumbrelocator.c:915]: Assignments should follow the order in which the struct fields are declared.

File: tests/core/arch-arm/stalker-arm.c [hunk 2]
```diff
@@ -1767,6 +1768,27 @@ TESTCASE (thumb_it_eq)
...
+);
+
+TESTCASE (thumb_it_al)
+{
+  INVOKE_THUMB_EXPECTING (GUM_NOTHING, thumb_it_al, 7);
+
+  g_assert_cmpuint (fixture->sink->events->len, ==, 0);
+}
+
+
```
Comment (oleavr) [inline tests/core/arch-arm/stalker-arm.c:1791]: Left-over blank line. (To match the existing style in frida-gum, blank lines should never follow each other.)

**Review by oleavr — CHANGES_REQUESTED** [review]
Thanks! 🤘

**Comment by WorksButNotTested** [comment]
Should be good to go.

**Pull request merged**